src/pages/home/report/EmojiPickerMenu/dynamicEmojiSize.js

```diff
@@ -0,0 +1,16 @@
+import styleVariables from '../../../../styles/variables';
+
+const dynamicEmojiSize = (windowWidth) => {
+    if (windowWidth <= 320) {
+        return styleVariables.fontSizeSmall;
+    }
+    if (
+        windowWidth <= 480
```

Comment:
Let's convert this to a single line if condition for consistency. Its weird seeing a mix of single and double line if statements when they have just 1 condition in them.